x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/repositories/encrypted/action/ChangeEncryptedRepositoryPasswordAction.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.repositories.encrypted.action;
+
+import org.elasticsearch.action.ActionType;
+import org.elasticsearch.xpack.core.repositories.encrypted.EncryptedRepositoryChangePasswordResponse;
+
+public final class ChangeEncryptedRepositoryPasswordAction extends ActionType<EncryptedRepositoryChangePasswordResponse> {
+
+    public static final String NAME = "cluster:admin/repository/change_password";
```

Comment:
Unsure whether we should name this `"cluster:admin/xpack/repository/change_password`. I think not, given the recent `cluster:admin/snapshot/mount`, and that the REST endpoints dropped the part.